freescout-resources/views/modules/enduserportal/partials/submit_form.blade.php

```diff
@@ -0,0 +1,252 @@
+                                                <option value="{{ $value }}" selected="selected">{{ $value }}</option>
+                                            @endforeach
+                                        </select>
+                                    @elseif ($custom_field->type == CustomField::TYPE_DROPDOWN_MULTISELECT)
+                                        {{-- Add hidden input to allow saving empty value --}}
+                                        <input type="hidden" name="cf[{{ $custom_field->id }}][]" value="" />
+                                        <select multiple="multiple" style="height:auto;" name="cf[{{ $custom_field->id }}][]"
+                                                class="form-control @if (!$custom_field->value) placeholdered @endif cf-multiselect-dropdown"
+                                                @if ($custom_field->required) required @endif placeholder="{{ $custom_field->name }}@if ($custom_field->required)*@endif">
+                                        @php
+                                            // Retrieve the selected values from the custom field
+                                            $selectedValues = is_array($custom_field->getMultiselectValues()) ? $custom_field->getMultiselectValues() : [];
+                                        @endphp
+                                        @if (is_array($custom_field->options))
+                                            @foreach ($custom_field->options as $option_key => $option_name)
+                                                <option value="{{ $option_name }}"
+                                                        {{ in_array($option_name, $selectedValues) ? 'selected' : '' }}>
+                                                    {{ $option_name }}
+                                                </option>
+                                            @endforeach
+                                        @endif
+                                        </select>
+                                    @else
+                                        <input name="cf[{{ $custom_field->id }}]" class="form-control eup-remember @if ($custom_field->type == CustomField::TYPE_DATE) eup-type-date @endif" value="{{ $custom_field_value }}" @if ($custom_field_value) data-prefilled="1" @endif placeholder="{{ $custom_field->name }}@if ($custom_field->required)*@endif" @if ($custom_field->required) required @endif
+                                            @if ($custom_field->type == CustomField::TYPE_NUMBER)
+                                                type="number"
+                                            @else
+                                                type="text"
+                                            @endif
+                                        />
+                                    @endif
+                                    <!-- TODO: Is this the right place for the help text? -->
```

Comment:
@rimi-itk Will you have a look here. I am not sure if it's the right place for the help-text.

Reply:
It is the right place, @yepzdk. https://github.com/itk-dev/freescout-docker/pull/8/commits/eba640c6d901ec418c94c2c4d421e7de0be4e5bf